### PR TITLE
Feat: Create Timer component + css w/ minutes and seconds displayed

### PR DESCRIPTION
**Pull request by Flumanuck**
I've made a timer that counts down from 2:00. You can also reset it while it's at zero.

### CODE REVIEW

File: client/src/Timer.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                    return 0;
+                });
+            }, 1000);
+        }
+    
+        return () => clearInterval(countdown);
+    };
+    
+    
+    const displayTime = (seconds) => {
```
Comment (J-Ariola) [inline client/src/Timer.jsx]: For a much clearer name, you can change `displayTime` to `displayTimerString`.

**Review by J-Ariola — COMMENTED** [review]
Please check the comments, but looks mostly good to me.
Move new components to a /src/components directory.
Don't forget to change any related imports and dependencies in App and other components if necessary.

File: client/src/Timer.jsx (hunk 1 above)
Comment (Flumanuck) [reply client/src/Timer.jsx]: I resolved this issue and renamed it as suggested

**Review by J-Ariola — APPROVED** [review]
LGTM

**Pull request merged**